src/store/metadata/stub/content/editions/versions/cpih_2022-01.json

```diff
@@ -11,37 +11,37 @@
             ],
             "@id": "https://staging.idpd.uk/datasets/cpih/editions/2022-01/versions/1",
             "identifier": "1",
-            "issued": "2017-01-01T00:00:00Z",
+            "issued": "2022-01-01T00:00:00Z",
             "title": "Consumer Prices Index including owner occupiers' housing costs (CPIH)",
-            "summary": "The Consumer Prices Index including owner occupiers' housing costs (CPIH) is a...",
-            "description": "The Consumer Prices Index...",
+            "summary": "CPIH is defined as the Consumer Prices Index including owner occupiers' housing costs. This reflects the fact that it is identical to the CPI, with the exception of inclusion of owner occupiers' housing costs and council tax.",
+            "description": "CPIH is the most comprehensive measure of inflation. It extends CPI to include a measure of the costs associated with owning, maintaining and living in one's own home, known as owner occupiers' housing costs (OOH), along with council tax.",
             "download_url": "https://staging.idpd.uk/datasets/cpih/editions/2022-01/versions/1.csv",
             "media_type": "text/csv",
             "table_schema": {
                 "columns": [
                     {
-                        "name": "Period",
-                        "datatype": "xsd:datetime",
-                        "titles": "Time period",
-                        "description": "The time period associated with the observation."
+                      "name": "period",
+                      "datatype": "datetime",
```

Comment:
Datatype doesn't match the dataset. It uses some combination of period/duration and date (P1Y/2012-01-01). Time is not used.

Reply:
Yes, I probably forgot to change that to "string".
Lots of details to check in trying to fix all the datasets in one go.